### PR TITLE
Enable live-restore capability in docker daemon

### PR DESCRIPTION
**Pull request by someword**
This change addresses issue #157

### CODE REVIEW

File: files/docker-daemon.json [hunk 1]
```diff
@@ -3,5 +3,6 @@
   "log-opts": {
     "max-size": "10m",
     "max-file": "10"
-  }
+  },
+  "no-new-privileges": true
```
Comment (mogren) [inline files/docker-daemon.json]: Sure this is the right change?  I would have expected `"live-restore": true`

`no_new_privs` might be a good idea anyway, but maybe a different issue?

Comment (someword) [reply files/docker-daemon.json]: ugh, my bad.  Let me fix

Comment (someword) [reply files/docker-daemon.json]: @mogren i've put in the correct change and rebased.
Sorry for the confusion :(

**Comment by dawidmalina** [comment]
LGTM. @micahhausler when this can be merged? Btw how often pull requests are reviewed and accepted (merged) or rejected (which is fine in some cases)? Most of the existing pull requests are not even reviewed :( those are not comfortable conditions encouraging to contributions (maybe that's why there are so many independent forks)

This process is not transparent or at least I don't see it :(

**Pull request merged**